akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventsourcedStashManagement.scala

```diff
@@ -17,23 +20,25 @@ private[akka] trait EventsourcedStashManagement[C, E, S] {
 
   def setup: EventsourcedSetup[C, E, S]
 
-  private def context = setup.context
+  private def context: ActorContext[InternalProtocol] = setup.context
 
   private def stashBuffer: StashBuffer[InternalProtocol] = setup.internalStash
 
   protected def stash(msg: InternalProtocol): Unit = {
+    if (setup.settings.logOnStashing) setup.log.debug("Stashing message: [{}]", msg)
 
-    val logLevel = setup.settings.stashingLogLevel
-    if (logLevel != Logging.OffLevel) context.log.debug("Stashing message: {}", msg) // FIXME can be log(logLevel once missing method added
-
-    val internalStashOverflowStrategy: StashOverflowStrategy = setup.persistence.defaultInternalStashOverflowStrategy
+    val internalStashOverflowStrategy: StashOverflowStrategy = {
```

Comment:
shouldn't this be done once, instead of for each message?
perhaps even put it in `setup`

Reply:
not fixed yet

Reply:
Missed this, thanks!